test/unit/containerDeploymentWebviewController.test.ts

```diff
@@ -501,6 +502,64 @@ suite("ContainerDeploymentWebviewController", () => {
         assert.equal(defaultResult.isDockerProfileValid, true);
     });
 
+    test("Test createConnectionGroup reducer", async () => {
+        const createConnectionGroupStub = sinon.stub(
+            ConnectionGroupWebviewController,
+            "createConnectionGroup",
+        );
+
+        createConnectionGroupStub.resolves("Error creating group");
+        let callState = (controller as any).state;

```

Comment:
Does `controller["state"]` work here?  That'd have the advantages of 1) getting compile-time validation that the property exists, and 2) typing callState correctly.